templates/map.js

```diff
@@ -12,109 +12,106 @@ const layersColoursDict = {
     hospitals: '#ff1111',
     diplomatic: '#446688',
     charityDropOff: '#11ee11',
-}
+};
 
-const LANG = '{{lang}}'
+const LANG = '{{lang}}';
 
 // we need to use url_for so flask freeze will include the file in build
 const dummyVariable = "{{ url_for('static', filename='style/layers.js') }}";
 
 function loadLayersVisibilitySave() {
-
     let state;
     if (typeof localStorage !== 'undefined') {
-
         state = localStorage.getItem('layers');
-        if (state) try {
-
-            state = JSON.parse(state);
-
-        } catch (e) { console.warn(e) }
+        if (state)
+            try {
+                state = JSON.parse(state);
+            } catch (e) {
+                console.warn(e);
+            }
     }
 
-    if(!state)
-        return {};
+    if (!state) return {};
 
     return state;
 }
 
 function getLayersState(layers, lang) {
-
     const definitions = {};
     const visibilities = loadLayersVisibilitySave();
 
-    layers.forEach(layer => {
-
+    layers.forEach((layer) => {
         const { group } = layer.metadata;
         const { visibility } = layer.layout;
 
         if (visibilities.hasOwnProperty(group)) {
-
-            layer.layout.visibility = (visibilities[group] ? 'visible' : 'none');
-
+            layer.layout.visibility = visibilities[group] ? 'visible' : 'none';
         } else if (!definitions[group]) {
             // takes state from definitions when not present
-            visibilities[group] = (visibility === 'visible');
+            visibilities[group] = visibility === 'visible';
         }
 
         if (definitions[group]) {
-
             definitions[group].layers.push(layer);
-
         } else {
-
             definitions[group] = {
                 layers: [layer],
                 id: group,
                 name: layer.metadata.name[lang],
-            }
+            };
         }
     });
 
     return {
         layersDefinitions: definitions,
-        layersVisibilityState: visibilities
+        layersVisibilityState: visibilities,
     };
 }
 
-const { layersDefinitions, layersVisibilityState } = getLayersState(layers.layers, LANG);
-const layersArray = Object.keys(layersDefinitions).map(id => layersDefinitions[id]);
+const { layersDefinitions, layersVisibilityState } = getLayersState(
+    layers.layers,
+    LANG
+);
+const layersArray = Object.keys(layersDefinitions).map(
+    (id) => layersDefinitions[id]
+);
 
 const sidebarDivId = 'sidebar-div';
 
 function loadCenter() {
-
     let center;
     if (typeof localStorage !== 'undefined') {
-
-        center = localStorage.getItem("center");
-        if (center) try {
-
-            center = JSON.parse(center);
-
-        } catch (e) { console.warn(e) }
+        center = localStorage.getItem('center');
+        if (center)
+            try {
+                center = JSON.parse(center);
+            } catch (e) {
+                console.warn(e);
+            }
```

Comment:
it would be nice to have rule of mandatory `{}` brackets for every if statement because that's very hard to read now